utils/open-vm-tools/Makefile

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright (C) 2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=open-vm-tools
+PKG_VERSION:=10.3.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/vmware/open-vm-tools.git
+PKG_SOURCE_VERSION:=stable-$(PKG_VERSION)
```

Comment:
Current most recent release version is 10.3.5.  Please also consider using the tarballs directly

Reply:
I checked the operation with 10.3.5, so I change the version.

release file name is stable-10.3.5.tar.gz.
I want to non change because the not include package name.
There is a danger of overwrite in the dl directory.
I want to leave the download as git.

Reply:
I mean the maintainer uploaded release tarball, like `open-vm-tools-10.3.5-10430147.tar.gz` at https://github.com/vmware/open-vm-tools/releases .  The git tag named tarballs are auto-generated by github and has no promise of consistent hashes across different downloads.

Reply:
fixed. use release tarball.